hoomd/md/pair/aniso.py

```diff
@@ -328,6 +328,65 @@ class ALJ(AnisotropicPair):
     only ``vertices`` creates a convex polytope (set ``vertices`` and ``faces``
     to empty lists to create the ellipsoid).
 
+    Important:
+        `ALJ` implicitly rounds the given shape by :math:`\sigma_c` via the WCA
+        or Lennard-Jones potential.
```

Comment:
It does not have the same effect as the rounding radius. The rounding radius changes the underlying shape of the particle from a convex polyhedron to a sphero-convex polyhedron. The contact distance *r* is computed between the two shapes.

I say that the contact sigma "implicltly" rounds the shape because the LJ potential has high energy values for *r* < 1.0 * contact sigma, preventing particles from achieving closer contacts in normal thermodynamic conditions.

Reply:
Resolved offline